src/Explorer/Controls/Editor/EditorReact.tsx

```diff
@@ -71,9 +86,14 @@ export class EditorReact extends React.Component<EditorReactProps, EditorReactSt
 
   protected configureEditor(editor: monaco.editor.IStandaloneCodeEditor) {
     this.editor = editor;
-    const queryEditorModel = this.editor.getModel();
     if (!this.props.isReadOnly && this.props.onContentChanged) {
-      queryEditorModel.onDidChangeContent(() => {
+      // Hooking the model's onDidChangeContent event because of some event ordering issues.
```

Comment:
This is when it is probably good to have a note as we are dependent on an undocumented order of events. So I like these comments here.